plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -186,8 +185,9 @@ def add_unit_metadata(self, item):
                     sum_element = ElementTree.SubElement(package_element, 'sum', sum_attributes)
                     sum_element.text = checksum_value
 
-                reboot_element = ElementTree.SubElement(package_element, 'reboot_suggested')
-                reboot_element.text = str(package.get('reboot_suggested', False))
+                if package.get('reboot_suggested', False):
```

Comment:
A nano suggestion. Probably there is no longer need for the default value in `get` since we now care only about the `True` case as above (line 90), and  just `package.get('reboot_suggested')` is enough.